packages/css/src/options.ts

```diff
@@ -25,7 +25,7 @@ export interface ThemeUIConfig {
   /**
    * Initializes the color mode based on the prefers-color-scheme media query
    */
-  useColorSchemeMediaQuery?: boolean
+  useColorSchemeMediaQuery?: 'system' | 'initial' | boolean
```

Comment:
```suggestion
  useColorSchemeMediaQuery?: 'system' | 'initial' | true /* same as 'initial' for compat */ | false
```

We can later update the docs to refer to `"initial"` instead of `true`.